flux/clusters/k8s-01/event-api-db/overlays/PROD/postgrescluster-instances.yaml

```diff
@@ -0,0 +1,36 @@
+# eventually it should be op: replace
+- op: add
+  path: /spec/instances/-
+  value:
+    name: hwn02 # see #19803
+    replicas: 1
+    resources:
+      limits:
+        memory: 512Gi # # let Postres use the page cache
+        cpu: '64' # set as event-api pool size + autovacuum workers + a bit for parallel workers
+        # empyrical recommendations below
+        # memory: 100Gi 
+        # cpu: '5' 
+      requests:
+        memory: 22Gi # roughly shared_buffers + pool size * work_mem
+        cpu: '2' # if we don't have CPU we can run on potato
+        # empyrical recommendations below
+        # memory: 7Gi
+        # cpu: '1' 
+    dataVolumeClaimSpec:
+      accessModes:
+        - ReadWriteOnce
+      resources:
+        requests:
+          storage: 100Gi
+          # empyrical recommendations below -- actual storage varies from 2300Gi to 6200Gi
+          # storage: 2500Gi
```

Comment:
That is fine, we use local-path-provisioner which de-facto ignores requested storage size, providing all available disk space.

As pod is anyway bound to the node it runs on, and the node disk usage may be significantly fluctuating, there's no practical benefit in specifying real requested size.

While I agree it may create confusion, I'd leave the decision to the code owners, to be sorted out later some day. Such a request size was here historically, so I am just following tradition (I can imagine how specifying higher values could create undesired disruptions in workloads scheduling).